blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/BasicMetricsRWIntegrationTest.java

```diff
@@ -38,15 +37,13 @@
     private static final String TENANT2 = "987654";
     private static final String TENANT3 = "123789";
 
-    protected LocatorIO locatorIO = new DLocatorIO();
-    protected DelayedLocatorIO delayedLocatorIO = new DDelayedLocatorIO();
+    protected DLocatorIO locatorIO = new DLocatorIO();
```

Comment:
I think this is ok. Only the DBasicMetricsRW class needs locatorIO and DelayedLocatorIO for its constructor. The ABasicMetricsRW class doesn't need those IO classes. (Don't ask me why, tho).  😬 Since DBasicMetricsRW is Datastax specific, it is ok to pass in Datastax specific IO classes to its constructor.